swf/src/types.rs

```diff
@@ -1382,6 +1692,12 @@ impl TextAlign {
     }
 }
 
+impl Default for TextAlign {
+    fn default() -> TextAlign {
+        TextAlign::Left
```

Comment:
We can use the now-stable https://rust-lang.github.io/rfcs/3107-derive-default-enum.html:
```rs
#[default]
Left
```